doc/nrf/gs_programming.rst

```diff
@@ -87,4 +87,91 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    	In a multi-image build, this allows you to debug the source code of your application only.
 
 If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
-If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
\ No newline at end of file
+If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
+
+.. _gs_programming_cmd:
+
+Building on the command line
+****************************
+
+Complete the following steps to build |NCS| projects on the command line after completing the command-line build setup (see the instructions for :ref:`Windows<build_environment_cli_win>`, :ref:`Linux<build_environment_cli_linux>`, or :ref:`macOS<build_environment_cli_mac>`).
+
+1.    Open a terminal window.
+
+#.    Go to the specific sample or application directory.
+      For example, to build the :ref:`at_client_sample` sample, run the following command to navigate to the sample directory:
+
+      .. code-block:: console
+
+         cd nrf/samples/nRF9160/at_client
+
+
+#.    Build the sample or application using the west command.
+      The development board is specified by the parameter *board_name* in the west command as follows:
+
+      .. parsed-literal::
+         :class: highlight
+
+         west build -b *board_name*
+
+      .. note::
+
+	     To build from a directory other than the sample or application directory, run the west build command with an additional parameter *directory_name*,  specifying the sample or application directory.
+
+      See `Board names <Board names_>`_ for more information on the development boards.
+      To reuse an existing build directory for building another sample or application for another board, pass ``-p=auto`` to ``west build``.
+
+      If you want to configure your application, run the following west command:
+
+      .. code-block:: console
+
+         west build -t menuconfig
+
+      See :ref:`configure_application` for additional information about configuring an application.
+
+      After running the ``west build`` command, the build files can be found in ``build/zephyr``.
+      For more information on the contents of the build directory, see the *Build Directory Contents* section in the Zephyr documentation on :ref:`zephyr:build_an_application`.
+
+      .. include:: gs_programming.rst
+         :start-after: .. imp_note_nrf91_start
+         :end-before: .. imp_note_nrf91_end
+
+#.    Connect the development board to your PC using a USB cable.
+#.    Power on the development board.
+#.    Program the sample or application to the board using the following command:
+
+      .. code-block:: console
+
+         west flash
+
+#.    Reset the board to start the sample or application.
```

Comment:
Additional note: If the board needs to be fully erased before programming the new app use `west flash --erase`

Also note `west flash` will reset the board automatically so no extra reset is required to start sample or application.